UQL/Interpreter.php

```diff
@@ -281,7 +283,9 @@ private function parseValue($value)
     private function callFunction(ASTFunctionCall $functionCall)
     {
         try {
-            return $this->extensionContainer->callFunction($functionCall->getFunctionName(), $functionCall->getArguments());
+            $context = new Context($this->dataSource->getEntityClass());
```

Comment:
instantiating new object within classes is not so cool... or is there a specific reason?